File: conditionals/time.py
# Lila Shearer, Time of Day Python

import time
#first instance of time in programming
first_time = time.gmtime() # This tells you what time it is in England 


#current time in seconds
current = time.time()
#print(current) #seconds since January 1 1970 because that's when computers started counting time.

#current date and time like we see it normally
now = time.ctime(current)
#print(now)

#get just a part of the time
local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour
print(day, hour)



#or you can do it like this
import time
current_time = time.localtime()  
hour = current_time.tm_hour  

File: conditionals/time.c
//Lila Shearer, How to do time in c
#include <stdio.h>
#include <time.h>

int main(void){
    //time since January 1, 1970
    time_t seconds;
    seconds = time(NULL);
    //printf("\nIt has been %d seconds since January 1st, 1970.\n", seconds);

    //now to get current time
    time_t rawtime;
    struct tm * timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
   // printf("Current time and date is %s.\n", asctime(timeinfo));
   
    //get current hour
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("The hour is %d.\n", hour);
   




    //or you can do it like this
    time_t t = time(NULL);
    struct tm *tm = localtime(&t);
    int hour = tm->tm_hour;
    return 0;
}